FirebasePerformance/Sources/FPRClient.m

```diff
@@ -107,6 +111,13 @@ - (instancetype)init {
     _eventsQueue = dispatch_queue_create("com.google.perf.FPREventsQueue", DISPATCH_QUEUE_SERIAL);
     _eventsQueueGroup = dispatch_group_create();
     _configuration = [FPRConfigurations sharedInstance];
+    NSString *plistPath = [[NSBundle mainBundle] pathForResource:@"GoogleService-Info"
+                                                          ofType:@"plist"];
+    NSMutableDictionary *googleServiceDict =
+        [NSMutableDictionary dictionaryWithContentsOfFile:plistPath];
+    _projectId = googleServiceDict[@"PROJECT_ID"];
```

Comment:
Instead of extracting this from the PLIST, can we extract this from [FIRApp sharedapp].options?

https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseCore/Sources/Public/FirebaseCore/FIROptions.h#L65